lighthouse-core/gather/driver.js

```diff
@@ -490,14 +488,53 @@ class Driver {
   }
 
   /**
-   * Navigate to the given URL. Use of this method directly isn't advised: if
+   * Set up listener for network quiet events and URL redirects. URL in options
+   * object will be updated to the post-redirected URL.
+   * @param {{url: string}} options
+   * @return {!Promise}
+   * @private
+   */
+  _beginNetworkMonitoring(options) {
```

Comment:
> rename to beginNetworkIdleMonitoring since its for busy idle detection. not to be confused with devtoolsLog like stuff?

we also watch for `requestloaded` to watch for url redirect. Not sure if that matters :)